### PR TITLE
Violin plot doc

### PR DESCRIPTION
**Pull request by priyatharsan**
closes https://github.com/plotly/plotly.github.io/issues/379

### CODE REVIEW

File: _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial-single_layout
+title: Violin Chart
```
Comment (cldougl) [inline _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md]: can we call this Violin Plot to stay consistent w/ the other documentation (ie https://plot.ly/python/violin/)

File: _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+subtitle: Violin Chart is a combination of a box plot and density plot that shows the distribution shape of the data.
```
Comment (cldougl) [inline _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md]: Violin Plot (throughout this whole doc)

File: _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md (hunk 1 above)
Comment (priyatharsan) [reply _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md]: 👍

File: _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md (hunk 2 above)
Comment (priyatharsan) [reply _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md]: :+1

Comment (priyatharsan) [reply _posts/statistical/2018-11-07-how-to-create-a-violin-chart.md]: 👍

**Pull request merged**